Packages/Status/Sources/Status/Row/StatusRowView.swift

```diff
@@ -453,43 +453,88 @@ public struct StatusRowView: View {
   }
 
   @ViewBuilder
-  private var trailinSwipeActions: some View {
-    Button {
-      Task {
-        HapticManager.shared.fireHaptic(of: .notification(.success))
+  private var trailingSwipeActions: some View {
+    if preferences.swipeActionsStatusRight1 != StatusAction.none {
+      makeSwipeButton(action: preferences.swipeActionsStatusRight1)
+        .tint(theme.tintColor)
+    }
+    if preferences.swipeActionsStatusRight2 != StatusAction.none {
+      makeSwipeButton(action: preferences.swipeActionsStatusRight2)
+        .tint(.gray)
+    }
+  }
+
+  @ViewBuilder
```

Comment:
Since we only have a single view being returned here, we can safely drop `@ViewBuilder`.

Reply:
If I remove these I seem to have to add explicit return statements which then lead to 
![image](https://user-images.githubusercontent.com/8456476/218271106-90c748a6-b30b-42b8-8677-c9bc30a1ab0d.png)